sonar-db/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -54,6 +54,7 @@ CREATE TABLE "GROUPS" (
 
 CREATE TABLE "SNAPSHOTS" (
   "ID" INTEGER NOT NULL GENERATED BY DEFAULT AS IDENTITY (START WITH 1, INCREMENT BY 1),
+  "UUID" VARCHAR(50) NOT NULL,
```

Comment:
in fact it already exists (see migration 1242) but it's missing in schema-h2.sql. I take the extract creation of index from 1242 to a dedicated migration.